nifi-nar-bundles/nifi-scripting-bundle/nifi-scripting-ui/src/main/resources/META-INF/NOTICE

```diff
@@ -0,0 +1,61 @@
+nifi-transform-json-ui
+Copyright 2014-2017 The Apache Software Foundation
+
+This product includes software developed at
+The Apache Software Foundation (http://www.apache.org/).
+
+******************
+Apache Software License v2
+******************
+
+The following binary components are provided under the Apache Software License v2
+
+  (ASLv2) Apache Commons Lang
+    The following NOTICE information applies:
+      Apache Commons Lang
+      Copyright 2001-2014 The Apache Software Foundation
+
+      This product includes software from the Spring Framework,
+      under the Apache License 2.0 (see: StringUtils.containsWhitespace())
+
+  (ASLv2) Jettison
+    The following NOTICE information applies:
+         Copyright 2006 Envoi Solutions LLC
+
+  (ASLv2) Jolt
```

Comment:
I assume this was copied from the Jolt UI package, but I believe can be removed here.

Reply:
Removed